src/MentionsTextInput.js

```diff
@@ -76,11 +76,16 @@ export default class MentionsTextInput extends Component {
   }
 
   resetTextbox() {
+    this.isResetting = true;
```

Comment:
When adding new member variables, it's a good practice to declare them above the constructor:

`isResetting: boolean;`

It's also a good practice to add declarations for existing members, time permitting.